packages/url-loader/src/plugins/cropping.ts

```diff
@@ -1,18 +1,23 @@
 import * as parameters from '../constants/parameters';
 
-import { PluginSettings, PluginOverrides } from '../types/plugins';
+import { PluginSettings, PluginResults } from '../types/plugins';
 
-const cropsAspectRatio = [ 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
-const cropsGravityAuto = [ 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
+const cropsAspectRatio = [ 'auto', 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
+const cropsGravityAuto = [ 'auto', 'crop', 'fill', 'lfill', 'fill_pad', 'thumb' ];
 const cropsWithZoom = ['crop', 'thumb'];
 
 const DEFAULT_CROP = 'limit';
 
 export const props = {
   aspectRatio: parameters.aspectRatio.schema.optional(),
+  baseAspectRatio: parameters.baseAspectRatio.schema.optional(),
+  baseCrop: parameters.baseCrop.schema.optional(),
+  baseGravity: parameters.baseGravity.schema.optional(),
+  baseHeight: parameters.baseHeight.schema.optional(),
+  baseWidth: parameters.baseWidth.schema.optional(),
+  baseZoom: parameters.baseZoom.schema.optional(),
   crop: parameters.crop.schema.default(DEFAULT_CROP).optional(),
   gravity: parameters.gravity.schema.optional(),
-  widthResize: parameters.widthResize.schema.optional(),
```

Comment:
question: should this actually be removed?

Reply:
yeah, the point is to move away from this pattern, where really its semantics, but the `widthResize` functionality intended for the value to be resized at the end, where now the `width` provides that capability. thats why `base` was added to provide the resizing at the front of the transformation chain

- widthResize => width (end of transformations)
- width => baseWidth (beginning of transformations)

`baseWidth/transformations/width`